web/src/assets/css/todo.css

```diff
@@ -310,8 +310,11 @@ h2 {
   padding: 0 8px;
 }
 
+.inactive {

```

Comment:
this should be specifically on `.skip span` and should be placed below the standalone `.skip span` style